commands/shitmemeboard.py

```diff
@@ -0,0 +1,242 @@
+from emoji import replace_emoji
```

Comment:
No, we do not copy paste large chunks of code like this. Parameterize the original.